### ISSUE TITLE
Bug with Tag Cloud css classes

### ISSUE THREAD

**Issue by hexstr1p** [radiac/django-tagulous#69]
When gathering the tags to create a tag cloud, it can be done like this
```python
tags = MyModel.tags.tag_model.objects.weight(min=2, max=4)
```
or this
```python
tags = MyModel.tags.tag_model.objects.weight()
```
Both of which _should_ create css classes in the view like `tag_1` or `tag_4` however, it is creating classes like `tag_1.0` and `tag_4.0`. 

Why is it adding `.0` to the tag weight? This happens if I pass in a min and max to the `weight()` or if I don't. Another weird thing is that this works as intended on a test server using an sqlite database, but consistently bugs adding a `.0` on another server using a postgres database, even though the tag gathering code is the same.

**Comment by hagsteel** [comment]
I've submitted a PR that might address this.
What might be helpful is to know:
*  Python version
*  Django version
*  Database (Postgres? MySql?)

**Comment by panpanick** [comment]
- Python version 3.5.2
- Django version 2.1.7
- Postgres database

**Comment by hagsteel** [comment]
I haven't been able to replicate this issue with tests.
In the meantime, could you create a property or a template tag to convert the float to an int?

**Comment by radiac** [comment]
This should be resolved by #70 which will be in the next release.
